.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every weekday
+      interval: "daily"
+    # Specify labels for pull requests
+    labels:
+      - "maintenance"
```

```suggestion
      - "maintenance"
      - "skip-changelog"
```
Maybe add skip-changelog as well?